splinepy/microstructure/microstructure.py

```diff
@@ -135,6 +135,11 @@ def microtile(self):
         if hasattr(self, "_microtile"):
             return self._microtile
         else:
+            self._logi(
```

Comment:
I don't know what your convention is here, but shouldn't this be a warning at least, as something was requested which seems to not be defined?

Reply:
`i`nfo, `d`ebug, `w`arning, `e`rror. @jzwar wanted this to be a friendly hint, and I thought info in friendlier than warning. I'd personally go with debug, since it is just unset value - however, per default, we have it off, so it won't be so helpful. do you still believe in `w`arning?

Reply:
I like info. As someone might use it in a Jupyter notebook first, without reading the documentation, ... This is a perfect hint.

Reply:
okay, I wasn't aware of the Jupyter notebook issue, I guess it is fine then. Most important is that the user is informed at all :)